### ISSUE TITLE
Questions about the Mean Field in CRF loss

### ISSUE THREAD

**Issue by LiWentomng** [NVlabs/mask-auto-labeler#11]
Hi, thanks for your wonderful work.

There is a question for me about the Mean Field in CRF loss.
In the below `trunc` function, the high threshold is set to 0.9. Could you please clarify the reason for this setting for the mean field process. When I  have changed 0.9 to 0.99, it even doesn't work.  Intuitively, 0.99 is  more precise than 0.9.  

```
    def trunc(self, seg):
        seg = torch.clamp(seg, min=self.low_thres, max=self.high_thres)
        return seg
```

**Comment by voidrank** [comment]
Hi @LiWentomng ,

Thanks for asking the details. I didn’t try too many combinations of the normalization/truncation function. As far as I remember, 0.1/0.9 are a pair of thresholds that are safe, but not necessary the best ones. The reason why it doesn’t work if you use a more precise threshold might be due to the log function used in the mean field algorithm.

**Comment by LiWentomng** [comment]
I see.  Thanks for your reply.
